packages/react/src/notifications/useNotifications.ts

```diff
@@ -30,17 +31,15 @@ export function useNotifications({
   profileId,
   limit,
 }: UseNotificationsArgs): PaginatedReadResult<Notification[]> {
-  const { apolloClient, sources } = useSharedDependencies();
-
   return usePaginatedReadResult(
-    useNotificationsQuery({
-      variables: {
-        observerId: profileId,
-        limit: limit ?? 10,
-        notificationTypes,
-        sources,
-      },
-      client: apolloClient,
-    }),
+    useNotificationsQuery(
+      useLensApolloClient({
+        variables: useConfigSourcesVariable({
+          observerId: profileId,
```

Comment:
why not use `useActiveProfileAsDefaultObserver` here?

Reply:
This is mostly in relation to the way this query was conceived. The assumption was that notifications are very personal. It's very unlikely that one asks someone else notifications from current active provide perspective. Probably even more so for asking your own notifications with someone else perspective.

So the query has already the request `profileId` also as the observer. Like saying, if you ask notification for profile A only profile A perspective matters.